src/main/java/com/openlattice/client/serialization/SerializationConstants.java

```diff
@@ -140,8 +145,11 @@
     public static final String INCLUDE_FIELD                    = "include";
     public static final String INDEXING                         = "indexing";
     public static final String INDEX_TYPE                       = "indexType";
+    public static final String IP_ADDRESSES                     = "ipAddresses";
+    public static final String IS_DESCENDING                    = "isDescending";
```

Comment:
this one had been renamed to just "descending" in a different PR and should stay that way due to weird kotlin side effects

Reply:
I'll delete it, it is showing as no longer used on my branch so should be safe